data/lua/wml/find_path.lua

```diff
@@ -77,10 +92,27 @@ function wesnoth.wml_actions.find_path(cfg)
 			utils.end_var_scope("this_unit", this_unit)
 		return end -- skip the cycles below
 
-		wesnoth.set_variable ( string.format( "%s", variable ), { hexes = current_distance, from_x = unit.x, from_y = unit.y, to_x = current_location[1], to_y = current_location[2], movement_cost = cost, required_turns = turns } )
+		wesnoth.set_variable (
+			string.format( "%s", variable ),
```

Comment:
I wonder what this thing is that I copy-pasted BTW. Is it some strange form of writing tostring(variable) ?

Reply:
Looks like a C programmer was learning Lua.

Reply:
Could as well be left as-is tho, doesn't really matter.

Reply:
Nah. Fix it. If this were C++ we'd be changing it to << on a string stream.

Reply:
done.